### PR TITLE
Add deterministic email sorting on an artwork

### PR DESCRIPTION
**Pull request by orta**
Ensures that because we don't retain the selection ordering, there is at least some sense of determinism in the order for the artworks Folio emails.

also, a lot of doc updates

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: ArtsyFolio Tests/Util/AREmailComposerTests.m [hunk 1]
```diff
@@ -446,6 +446,24 @@ @interface AREmailComposer ()
             expect(body).toNot.contain(@"https://www.artsy.net/artwork/ThisShouldBeInHere");
         });
     });
+
+    it(@"Sorts by artist dispaly then artwork title", ^{
```
Comment (jonallured) [inline ArtsyFolio Tests/Util/AREmailComposerTests.m]:
```suggestion
    it(@"Sorts by artist display then artwork title", ^{
```

File: ArtsyFolio Tests/Util/AREmailComposerTests.m [hunk 2]
```diff
@@ -446,6 +446,24 @@ @interface AREmailComposer ()
...
+        Artwork *artwork = [Artwork objectInContext:context];
```
Comment (jonallured) [inline ArtsyFolio Tests/Util/AREmailComposerTests.m]: I would suggest calling this one `artwork1` to follow the pattern of the others. #minor

File: ArtsyFolio Tests/Util/AREmailComposerTests.m [hunk 3]
```diff
@@ -446,6 +446,24 @@ @interface AREmailComposer ()
...
+        artwork2.artistOrderingKey = @"Aaron";
+        artwork2.displayTitle = @"Artwork 2";
+
+        Artwork *artwork3 = [Artwork objectInContext:context];
+        artwork3.artistOrderingKey = @"Barr";
+        artwork3.displayTitle = @"Artwork 1";
+
+        composer.artworks = @[ artwork3, artwork2, artwork ];
+
+        expect(composer.artworks).to.equal(@[ artwork, artwork2, artwork3 ]);
```
Comment (jonallured) [inline ArtsyFolio Tests/Util/AREmailComposerTests.m]: Nice test! 😎

**Comment by orta** [comment]
merge on green

**Comment by orta** [comment]
Poor old Peril.

**Pull request merged**